src/clients/api/queries/getMainMarkets/index.ts

```diff
@@ -4,38 +4,47 @@ import { restService } from 'utilities';
 
 export interface ApiMarket {
   address: string;
-  totalDistributed: string;
+  totalDistributedMantissa: string;
+  underlyingDecimal: number;
   borrowerCount: number;
   supplierCount: number;
 }
 
 export interface GetMainMarketsResponse {
-  dailyVenus: number;
-  markets: ApiMarket[];
+  result: ApiMarket[];
   request: { addresses: string[] };
-  venusRate: string;
 }
 
 export interface GetMainMarketsOutput {
   markets: Market[];
 }
 
 const getMainMarkets = async (): Promise<GetMainMarketsOutput> => {
-  const response = await restService<GetMainMarketsResponse, 'v1'>({
+  const response = await restService<GetMainMarketsResponse, 'v2'>({
     endpoint: '/markets/core-pool',
     method: 'GET',
+    next: true,
```

Comment:
Shouldn't we use the "stable" version of the backend in production?
I'm wondering whether we put ourselves at risk of crashing the frontend if a new "next" version of the backend is deployed and we haven't made the changes yet.

Reply:
It is intended to use next for now, as long as we follow the laid out plan:

1. move frontend to `next` as in this PR
2. backend promotes `next` to `stable` while keeping the current next also valid
3. frontend turns off `next`
4. backend is free to implement a new `next`